responsive-web-design/css-typography/index05.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <body>
+    <h1>Nutrition Facts</h1>
+    <p>8 servings per container</p>
+    <p>Serving size 2/3 cup (55g)</p>
+  </body>
+</html>
+
+<!-- 
+You'll need CSS to use your new typeface.
+Create a style block between the head and body elements.
+Add an opening `<style>` tag right below the closing `</head>` tag, and a closing `</style>` tag above the opening `<body>` tag.
+-->
+<!-- 
+Note: This and the following steps will need to be changed since the plan is to have the user work with multiple files.
+Could change the lesson to one/several about using a relative path to link to a CSS file.
+-->
```

Comment:
Imo because this lesson deals with html header and paragraph tags I'm okay with leaving it in the HTML file (we can teach them `style.css` linking in another challenge set. 

I am also in favor of the multi file approach, but until we know what that lesson experience will be like its best to keep things in one file if we can.